Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -618,7 +631,7 @@ std::vector<double> MaxEnt::applyDistancePenalty(
   auto newDelta = delta;
   if (dist > distEps * sum / background) {
     for (size_t k = 0; k < delta.size(); k++) {
-      newDelta[k] *= sqrt(sum / dist / background);
+      newDelta[k] *= sqrt(distEps * sum / dist / background);
```

Comment:
why have you added a distEps?

Reply:
If `dist` is too large you want to make `delta` smaller. The factor is the solution of the equation
```
factor^2 * dist == distEps * sum / background
```
After applying this factor the above `if` statement would fail if tried again.